docs/core/versions/versioning-standard.md

```diff
@@ -0,0 +1,21 @@
+---
+Your magic stuff
+---
+
+# Versioning .NET Standard
+
+.NET Standard consists of a reference library and implementations specific to each platform. The reference library contains the definition of .NET Standard. Each implementation fulfills the .NET Standard contract on the specific platform. .NET Standard usually refers to the reference library, and the implementation is detail of the corresponding runtime.
```

Comment:
"reference library" is a troubling term. It could be read (mistakenly) as a reference implementation. I'd add a clarifying sentence that the .NET Standard reference library is a ".NET Reference Assembly", meaning there is API definitions but no implementation.